sunbeamlib/initialize.py

```diff
@@ -42,8 +45,15 @@ def main():
     else:
         args.conda_fp = args.conda_fp.absolute()
 
-    config = create_blank_config(
-        args.conda_fp, args.project_fp, template=args.server)
-    
+    if args.custom_configfile_fp is None:
+        config = create_blank_config(
+            args.conda_fp, args.project_fp, template=args.server)
+    elif os.path.isfile(args.custom_configfile_fp.name):
```

Comment:
`args.custom_configfile_fp` is already an open file handle to an existing file if the `parser.parse_args` function returned successfully, so this check is unnecessary -- if the file doesn't exist, the error will be generated by argparse in line 38. Please simplify to:
```python
else:
  config = args.custom_configfile_fp
```